azkaban-common/src/main/java/azkaban/executor/container/watch/FlowStatusManagerListener.java

```diff
@@ -283,11 +288,12 @@ private void deleteFlowContainer(AzPodStatusMetadata event) {
 
   /**
    * Finalize the flow for the given event and delete its container.
-   *
    * @param event
+   * @param finalStatus
    */
-  private void finalizeFlowAndDeleteContainer(AzPodStatusMetadata event) {
-    Optional<Status> originalFlowStatus = compareAndFinalizeFlowStatus(event);
+  private void finalizeFlowAndDeleteContainer(AzPodStatusMetadata event,
```

Comment:
This PR implements the new status for containerized executions only but bare metal executors also crash so we should be consistent and implement it for bare metal executors too. Could you please make the necessary changes?
So far execution failures due to executor crashes has been marked as FAILED. When this PR is merged their final status should change to EXECUTION_STOPPED which affects ALL Azkaban users and potentially break their code. Users will need to modify their code and logic to account for the new status. Are we ok with that? @sshardool @jakhani @djaiswal83 @aditya1105